### ISSUE TITLE
Error with fresh install of latest

### ISSUE THREAD

**Issue by deprekate** [JuliaEditorSupport/julia-vim#197]
Error on fresh installed on OS X 10.11.6 via manual
```
git clone git://github.com/JuliaEditorSupport/julia-vim.git
cd julia-vim
mkdir -p ~/.vim
cp -R * ~/.vim
```
also your manual instructions are terrible, since it dumped the entire repo into my base .vim directory (INSTALL, LICENSE, README, logo.png, etc) and while going through deleting the bad install I accidentally deleted something that wasn't part of your repo.  It's like when people tar/zip a ton of files without a nested folder, just bad form

:version                                                                                                                                            
VIM - Vi IMproved 7.3 (2010 Aug 15, compiled Sep 26 2017 10:21:06)

I get the following wall of errors
```
"hello.jl" [New File]
Error detected while processing /Users/katelyn/.vim/syntax/julia.vim:
line  127:
E59: invalid character after \@
E475: Invalid argument: juliaKeyword^I^Idisplay "\%(\.\)\@1<!\<\%(return\|local\|global\|import\%(all\
)\?\|export\|using\|const\|where\)\>"
line  129:
E59: invalid character after \@
E475: Invalid argument: juliaRepKeyword^I^Idisplay "\%(\.\)\@1<!\<\%(break\|continue\)\>"
line  130:
E59: invalid character after \@
E475: Invalid argument: juliaConditionalBlock^Imatchgroup=juliaConditional start="\%(\.\)\@1<!\<if\>"
end="\%(\.\)\@1<!\<end\>" contains=@juliaExpressions,juliaConditionalEIBlock,juliaConditionalEBlock fo
ld
line  131:
E59: invalid character after \@
E475: Invalid argument: juliaConditionalEIBlock^Imatchgroup=juliaConditional transparent contained sta
rt="\%(\.\)\@1<!\<elseif\>" end="\%(\.\)\@1<!\<\%(end\|else\|elseif\)\>"me=s-1 contains=@juliaExpressi
ons,juliaConditionalEIBlock,juliaConditionalEBlock
line  132:
E59: invalid character after \@
E475: Invalid argument: juliaConditionalEBlock^Imatchgroup=juliaConditional transparent contained star
t="\%(\.\)\@1<!\<else\>" end="\%(\.\)\@1<!\<end\>"me=s-1 contains=@juliaExpressions
line  133:
E59: invalid character after \@
-- More --
```

**Comment by carlobaldassi** [comment]
Yes those manual installation instructions were very old, probably written when the plugin barely had a syntax file and some indentation rules. I have now revised the page, thanks for pointing this out.

As for the errors that you get, those should not really occur, so it's perplexing. The error comes from trying to use things like `\@1<!` in regular expressions, which are not supported by your version of vim. However this issue had already been solved: #86.

Can you check that you actually have [these lines](https://github.com/JuliaEditorSupport/julia-vim/blob/3527291dda44f401160aff5b9c710a34c6392c74/syntax/julia.vim#L12-L22) in your installed version? That should detect your vim version, and if it isn't we need to figure out why. Is it possible that problems during the installation mean that you have an older version of the plugin installed?

**Comment by deprekate** [comment]
>Yes those manual installation instructions were very old, probably written when the plugin barely had a syntax file and some indentation rules. I have now revised the page, thanks for pointing this out.

Cool, but you should really have at least a minimal instruction set for people who don't have, or want to install, a third party program just to get julia syntax highlighting

<br>

> Can you check that you actually have these lines in your installed version?

Yep, my syntax/julia.vim has those lines

<br>

> Is it possible that problems during the installation mean that you have an older version of the plugin installed?

Yesterday was the first time I ever installed julia-vim 

<br>

The only things in my .vim directory are the seven julia-vim directories and here's the contents of my .vimrc (nothing conflicts)
```
filetype plugin indent on
syntax on
set ruler
set hlsearch
set scrolloff=5
if has("autocmd")
        au BufReadPost * if line("'\"") > 0 && line("'\"") <= line("$") | exe "normal! g`\"" | endif
endif
```

**Comment by deprekate** [comment]
I just copied the older version of sytax/julia.vim and it works fine
https://github.com/JuliaEditorSupport/julia-vim/blob/22a100d4fcbbd98e74ecff9c48fe29119198dca3/syntax/julia.vim

So it seems to most likely be the s:nodot versus s:d stuff

**Comment by carlobaldassi** [comment]
I'm an idiot :)
I had already tried to fix this particular problem in #170 but I had received no feedback and I had assumed it had worked. Clearly, it had not. Hopefully it works now. Otherwise please let me know and I'll reopen the issue.

As for the manual installation instructions, really the best advice I can give for a plugin with as much stuff as this one is to rely on some package manager, be it another plugin or a built-in one which comes with more recent versions of vim. Also, as you've seen, it's very easy for the kind of detailed information that is needed to go out of sync, since I think the vast majority of people relies on plugin managers and nobody reports problems. Also having a git-based manager helps keeping things up to date very easily, and reverting easily in case of breakage. I have looked around a bit and other similar plugins use more or less the same wording in their readme for manual installation instructions.

**Comment by deprekate** [comment]
awesome, the updated one works perfect, and I even have the correct highlighting for 'export' and 'struct' (the #86 version wasn't highlighting them).  Thanks for the quick resolution
